src/main.py

```diff
@@ -217,6 +227,14 @@ async def task_add_unupdated_role():
 
     print("done")
 
+@tasks.loop(time=time(hour=23, minute=45))
```

Comment:
personally I use the `black` autoformatter as that seems to be the standard for python, not strict about it or anything though. I personally like the convenience of autoformatters but I know some people have mixed opinions